apps/web/src/actions/commits/index.ts

```diff
@@ -0,0 +1,19 @@
+'use server'
+
+import { isCommitDraft } from '@latitude-data/core'
+import { z } from 'zod'
+
+import { authProcedure } from '../procedures'
+
+export const checkCommitEditableAction = authProcedure
+  .createServerAction()
+  .input(
+    z.object({
+      commitUuid: z.string(),
+    }),
+    { type: 'json' },
+  )
+  .handler(async ({ input }) => {
+    const res = await isCommitDraft({ uuid: input.commitUuid })
+    return res.unwrap()
+  })
```

Comment:
does this need to be a server action? you are gonna need the list of commits in the frontend regardless so you can check there is a commit is draft or not